source/testnet/desktop-wallet/get-started-desktop.rst

```diff
@@ -0,0 +1,41 @@
+.. _get-started-desktop:
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 2
+
+====================================
+Get started with the Desktop Wallet
+====================================
+
+The Concordium Desktop Wallet is a digital wallet that enables you to create and manage your Concordium identities, credentials, and accounts, and to create transactions such as sending GTU, adding a baker, and exporting and importing account information.
+
+The Desktop Wallet also supports storing of your private keys on a hardware wallet. Currently, only the Ledger Nano S is supported.
+
+Prerequisites
+=============
+
+-  :ref:`Run a node <run-node-ubuntu>` on the Concordium blockchain.
```

Comment:
Could we please have some explanation of why you need to run a node. e.g. The Desktop Wallet needs access to a node to interact with the Concordium Blockchain (i.e. getting updates and posting transactions)